lib/common/declarations.d.ts

```diff
@@ -1143,6 +1143,12 @@ interface IDeviceLiveSyncService extends IDeviceLiveSyncServiceBase {
 	afterInstallApplicationAction?(deviceAppData: Mobile.IDeviceAppData, localToDevicePaths: Mobile.ILocalToDevicePathData[]): Promise<boolean>;
 }
 
+interface ISystemWarning {
+	message: string;
+	severity: SystemWarningsSeverity;
+	toString?: () => string;
```

Comment:
Forgot to mention this in the PR description - the `toString` is required for a minimal backwards compatibility. With the old code, you were able to use the result of `getSystemWarnings()` and join it with EOL for example and print it. In case we do not have the `toString` implementation here, the join operation will use the default toString, which returns `[Object object]`.